app/helpers/application_helper.rb

```diff
@@ -16,6 +16,13 @@ def tooltip(text)
     )
   end
 
+  def alert_category(key)
+    key = key.to_s.downcase
+    return 'oops' if key.in? %w(error alert)
+    return 'success' if key.in? %w(notice success)
```

Comment:
same comment here -- I assume we own all the code that creates theses flash messages. would it make sense to just updated them to all be `notice` or `success` rather than forcing them to be the same via this helper?